### PR TITLE
refactor: rename to OpaquePrompts

### PR DESCRIPTION
**Pull request by zizhong**
Renamed to OpaquePrompts

cc @baskaryan Thanks in advance!

### CODE REVIEW

**Review by leo-gan — APPROVED** [review]
LGTM

File: docs/extras/integrations/llms/opaqueprompts.ipynb [hunk 1]
```diff
@@ -172,15 +172,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import langchain.utilities.promptguard as pgf\n",
+    "import langchain.utilities.opaqueprompts as op\n",
```
Comment (leo-gan) [inline docs/extras/integrations/llms/opaqueprompts.ipynb:175]: why don't use this directly
`from langchain.utilities.opaqueprompts import desanitize`

and L193 `op.desanitize..` -> `desanitize...`

Comment (zizhong) [reply docs/extras/integrations/llms/opaqueprompts.ipynb:175]: We could. `import as` offers a slightly better namespace I think.

Comment (leo-gan) [reply docs/extras/integrations/llms/opaqueprompts.ipynb:175]: It is just about a sideeffect. If we use `import langchain.utilities.promptguard as pgf` the doc build does not include the `desanitize` in the` API Reference` section. Here is an example https://python.langchain.com/docs/integrations/providers/ainetwork

Comment (zizhong) [reply docs/extras/integrations/llms/opaqueprompts.ipynb:175]: Oh I see. Didn't notice that. Yes, I can change it that way. Thanks for the insights!

**Pull request merged**